dask/dataframe/io/parquet/fastparquet.py

```diff
@@ -19,9 +20,23 @@
 except ImportError:
     pass
 
-from .utils import _parse_pandas_metadata, _normalize_index_columns, _analyze_paths
+from .utils import (
+    _parse_pandas_metadata,
+    _normalize_index_columns,
+    _analyze_paths,
+    _flatten_filters,
+    _row_groups_to_parts,
+)
 from ..utils import _meta_from_dtypes
 from ...utils import UNKNOWN_CATEGORIES
+from ...methods import concat
+
+try:
+    # Required for distrubuted
+    import pickle
```

Comment:
Ah good question - I was assuming that `pickle` is a dependency for `distributed` but not for `dask`, but never checked this.

Reply:
Where can I find the list of official dependencies for Dask?  If `pickle` is an official dependency, we can remove this check.  Otherwise, I'd like to keep it.

Reply:
pickle is in the stdlib
https://docs.python.org/3.8/library/pickle.html

Reply:
ah - so it is :)